pygments/lexers/lisp.py

```diff
@@ -2670,7 +2675,6 @@ class FennelLexer(RegexLexer):
             (r'-?\d+', Number.Integer),
 
             (r'"(\\\\|\\"|[^"])*"', String),
```

Comment:
Sorry, I just saw this right now: this regex is prone to catastrophic backtracking, we should fix it while at it.  It should look like 
```
"(\\\\|\\"|[^"\\])*"
```
with alterations as needed for other allowed escape sequences.  If they are complex, a separate state can be used as well.

Reply:
That's literally the third time in a week or so we got the same regex of doom :)

Reply:
Oh, well really that regex was added years ago; copied from an existing lexer IIRC. But I'm glad to see it gone.